web/src/main/resources/META-INF/resources/webjars/common/webcert/utils/http403ResponseInterceptor.service.js

```diff
@@ -31,55 +31,58 @@
  */
 angular.module('common').provider('common.http403ResponseInterceptor',
     function() {
-        'use strict';
+      'use strict';
```

Comment:
Is it possible to make your changes without changing the code format? Now the indentation seems to have changed.